src/js/modules/swiperHelpers.js

```diff
@@ -51,4 +51,26 @@ export function loadAllSliderImages(sectionClass) {
 	}, {rootMargin: "0px 0px 200px"});
 
 	observer.observe(section);
+}
+
+export function disableFocusInInactiveSlides(swiper, swiperContainerSelector) {
```

Comment:
it is shit code

use "inert" attribute with active or inactive classes or look at api, maybe there is something about accessibility or blocking focus

google how to block focus with css

Reply:
Done